rpcs3/Emu/perf_monitor.cpp

```diff
@@ -20,10 +20,11 @@ void perf_monitor::operator()()
 	u64 last_pause_time = umax;
 
 	std::vector<double> per_core_usage;
+	std::string msg;
 
-	while (thread_ctrl::state() != thread_state::aborting)
+	for (u64 sleep_until = get_system_time(); thread_ctrl::state() != thread_state::aborting;)
 	{
-		thread_ctrl::wait_for(update_interval_us);
+		thread_ctrl::wait_until(&sleep_until, update_interval_us);
```

Comment:
Same in this file

Reply:
It's a good place to demonstrate the use of the function because it is expected that the intervals are consistent here.

Reply:
Is it though? It's just a random log message.

Reply:
Anything performance-measurement related is expected to be as precise as possible with timings, it's also a small and clean file so it better conveys the intent.